### DISCUSSION TITLE
Key binding to clear current line content

### DISCUSSION THREAD

**Discussion by 1lessbug** [fish-shell/fish-shell#9178] [Q&A] [answered]
I wanted to have key binding to clear the current line using COMMAND + DELETE key but nowhere I can find character sequence to represent command key. On [this](https://fishshell.com/docs/current/cmds/bind.html) doc, I see \c is for control, but what about Command and Delete?
I know terminal has control + U to do this, but I'm used to command + delete

**Comment by faho** [comment, marked as answer]
Run `fish_key_reader`, press whatever key sequence you want, and it will tell you the character sequence.

Note that many sequences are not representable, e.g. there is no way for fish to tell ctrl-i from tab. That's unfortunately just how terminals operate.

**Comment by floam** [comment]
You aren't going to be able to bind any command-modified keys in Terminal.app on macOS.
